src/sourmash/picklist.py

```diff
@@ -43,51 +43,85 @@ class SignaturePicklist:
+                # for now, override => md5short in column md5
+                coltype = 'md5prefix8'
+                column_name = 'md5'
+            elif coltype == 'prefetch':
+                # for now, override => md5short in column match_md5
+                coltype = 'md5prefix8'
+                column_name = 'match_md5'
+            elif coltype == 'manifest' or coltype == 'search':
+                # for now, override => md5
+                coltype = 'md5'
+                column_name = 'md5'
+            else:               # should never be reached!
+                assert 0
+
         self.coltype = coltype
         self.pickfile = pickfile
         self.column_name = column_name
         self.pickstyle = pickstyle
 
-        if coltype not in self.supported_coltypes:
-            raise ValueError(f"invalid picklist column type '{coltype}'")
-
-
         self.preprocess_fn = preprocess[coltype]
         self.pickset = None
         self.found = set()
         self.n_queries = 0
 
     @classmethod
     def from_picklist_args(cls, argstr):
-        "load a picklist from an argument string 'pickfile:column:coltype'"
+        "load a picklist from an argument string 'pickfile:col:coltype:style'"
         picklist = argstr.split(':')
-        if len(picklist) != 3:
-            if len(picklist) == 4:
-                pickfile, column, coltype, pickstyle = picklist
-                if pickstyle == 'include':
-                    return cls(coltype, pickfile=pickfile, column_name=column, pickstyle=PickStyle.INCLUDE)
-                elif pickstyle == 'exclude':
-                    return cls(coltype, pickfile=pickfile, column_name=column, pickstyle=PickStyle.EXCLUDE)
-                else:
-                    raise ValueError(f"invalid picklist 'pickstyle' argument, '{pickstyle}': must be 'include' or 'exclude'")
+        pickstyle = PickStyle.INCLUDE
+
+        # pickstyle specified?
+        if len(picklist) == 4:
+            pickstyle_str = picklist.pop()
+            if pickstyle_str == 'include':
+                pickstyle = PickStyle.INCLUDE
+            elif pickstyle_str == 'exclude':
+                pickstyle = PickStyle.EXCLUDE
+            else:
+                raise ValueError(f"invalid picklist 'pickstyle' argument, '{pickstyle_str}': must be 'include' or 'exclude'")
 
+        if len(picklist) != 3:
             raise ValueError(f"invalid picklist argument '{argstr}'")
 
         assert len(picklist) == 3
         pickfile, column, coltype = picklist
 
-        return cls(coltype, pickfile=pickfile, column_name=column)
+        return cls(coltype, pickfile=pickfile, column_name=column,
+                   pickstyle=pickstyle)
 
     def _get_sig_attribute(self, ss):
         "for a given SourmashSignature, return attribute for this picklist."
         coltype = self.coltype
-        if coltype in ('md5', 'md5prefix8', 'md5short'):
+        if coltype in ('md5', 'md5prefix8', 'md5short', 'manifest', 'gather'):
```

Comment:
should prefetch and search also be in here?

Reply:
actually, no - they should all be removed! (for now, the various meta types just pick md5sum, so the coltype becomes md5/md5short). I removed this change in 567d1708.

nice catch :)